notebooks/clean-retraining/SAMO_Stage1_GoEmotions_Foundation.ipynb

```
@@ -0,0 +1,1128 @@
+        "        'macro_f1': f1_macro.item(),\n",
+        "        'micro_precision': precision_micro.item(),\n",
+        "        'micro_recall': recall_micro.item(),\n",
+        "        'macro_precision': precision_macro.item(),\n",
+        "        'macro_recall': recall_macro.item(),\n",
+        "        'subset_accuracy': subset_accuracy.item(),\n",
+        "        'hamming_accuracy': hamming_accuracy.item(),\n",
+        "        'sample_f1_mean': f1.mean().item(),\n",
+        "        'threshold': threshold\n",
+        "    }\n",
+        "\n",
+        "def evaluate_model(model, data_loader, device):\n",
```

Ruff (0.12.2)

`print` found

Remove `print`

(T201)